From 13b503dbbf8c8ba779c599b8f56bed915264aef2 Mon Sep 17 00:00:00 2001
From: Hansong Zhang <hsz@google.com>
Date: Thu, 16 Aug 2018 09:46:45 -0700
Subject: [PATCH 18/20] Fix build failure in stack/rfcomm/rfc_ts_frames.c

Test: compile
Bug: 112673718
Change-Id: I93cd39f943dd2f0fb65b785c15dc91649c7ee384
(cherry picked from commit eb3e2528714bd6ea59ad369798f522d75a2e55c7)
---
 stack/rfcomm/rfc_ts_frames.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/stack/rfcomm/rfc_ts_frames.cc b/stack/rfcomm/rfc_ts_frames.cc
index 8e7c11849..8b21e3cfe 100644
--- a/stack/rfcomm/rfc_ts_frames.cc
+++ b/stack/rfcomm/rfc_ts_frames.cc
@@ -26,6 +26,7 @@
 #include "bt_common.h"
 #include "bt_target.h"
 #include "l2c_api.h"
+#include "log/log.h"
 #include "port_api.h"
 #include "port_int.h"
 #include "rfc_int.h"
-- 
2.17.1

